pkg/daemon/manager/manager.go

```diff
@@ -39,6 +41,14 @@ func (m *manager) OpenService(name string) (winsvc.Service, error) {
 	underlyingMgr := (*mgr.Mgr)(m)
 	return underlyingMgr.OpenService(name)
 }
+func (m *manager) CloseService(name string) error {
```

Comment:
PS: No need to bunch these functions together. Feel free to add new lines.

Reply:
A some whitespace :)